Tasks/DownloadPackageV1/task.json

```diff
@@ -9,8 +9,8 @@
     "author": "ms-vscs-rm",
     "version": {
         "Major": 1,
-        "Minor": 165,
```

Comment:
please update to Minor 174 Patch 0

Reply:
Please update it now to 176 and commit before the end of the sprint

Reply:
done